kolibri/deployment/default/wsgi.py

```diff
@@ -6,11 +6,10 @@
 For more information on this file, see
 https://docs.djangoproject.com/en/1.8/howto/deployment/wsgi/
 """
-
-import os
-
 from django.core.wsgi import get_wsgi_application
 
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "kolibri.deployment.default.settings.base")
+from kolibri.utils.cli import initialize
+
+initialize()
```

Comment:
Feels maybe a bit weird importing something from cli to use here, but didn't want to do a huge refactor of that code at this stage. Plus, it does what we want!